### PR TITLE
Feature/19 content testing issues

### LINKED ISSUE #19: Guides - issues found in initial content testing
1) On landing page eg https://localgovdrupal.agile.coop/adult-health-and-social-care/travel-passes-and-support, the incorrect summary is being shown. Should be seeing content in [Guide description] and not the first xx chars of Body

2) The grey links panel should include a link to the overview, and use [Guide section title] as the wording

3) Guide pages should use the title of the Guide overview [title] as the title. See https://new.croydon.gov.uk/adult-health-and-social-care/travel-passes-and-support/blue-badges/blue-badges-organisations for an example

### PR DESCRIPTION
**Pull request by stephen-cox**
* Minor display change to overview teasers
* Added overview title to contents block

Closes #19

### CODE REVIEW

**Comment by stephen-cox** [comment]
I believe this is also closes #9 . My comment on that issues now refers to this issue: https://github.com/localgovdrupal/localgov/issues/87

File: src/Plugin/Block/GuidesContentsBlock.php [hunk 1]
```diff
@@ -23,8 +21,9 @@ public function build() {
     $this->setPages();
     $links = [];
 
+    $options = $this->node->id() == $this->overview->id() ? ['attributes' => ['class' => 'active']] : [];
+    $links[] = $this->overview->toLink($this->overview->localgov_guides_section_title->value, 'canonical', $options);
     foreach ($this->guidePages as $guide_node) {
-      assert($guide_node instanceof NodeInterface);
```
Comment (ekes) [inline src/Plugin/Block/GuidesContentsBlock.php:27]: This was there as a type hind, equivalent of `/** @var \Drupal\node\NodeInterface $guide_node */` except actually PHP standard and correct - and if it explodes you know you've got something very broken.

Comment (ekes) [reply src/Plugin/Block/GuidesContentsBlock.php:27]: As I was already in the coding standards queue: https://www.drupal.org/project/coding_standards/issues/2425219#comment-12848176

Comment (stephen-cox) [reply src/Plugin/Block/GuidesContentsBlock.php:27]: I've restores the missing NodeInterface assertion.

**Pull request merged**